web/src/components/Upstream/components/active-check/Healthy/HttpStatuses.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react'
+import { Form, Row, Col, Button, InputNumber } from 'antd'
+import { useIntl } from 'umi'
+import { MinusCircleOutlined, PlusOutlined } from '@ant-design/icons'
+
+import { removeBtnStyle } from '@/components/Upstream'
+
+type Props = {
+  readonly?: boolean
+}
+
+const Component: React.FC<Props> = ({ readonly }) => {
+  const { formatMessage } = useIntl()
+
+  return (
+    <Form.List name={['checks', 'active', 'healthy', 'http_statuses']} initialValue={[200, 302]}>
+      {(fields, { add, remove }) => (
+        <>
+          <Form.Item
+            required
```

Comment:
Does the form validation work when written this way?

Reply:
all those options are optional, only for style effect.

![image](https://user-images.githubusercontent.com/2106987/114651435-8a4bd900-9d16-11eb-845d-26b54358f3cc.png)